js/index.js

```diff
@@ -38,12 +39,58 @@ window.addEventListener('load', event => {
 
   // Add all the divs to the HTML
   document.querySelector('#memory-board').innerHTML = html;
-
+  //pairsClickedHTML = document.querySelector('#pairs-clicked')
+  //pairsGuessedHTML = document.querySelector('#pairs-guessed')
+  
   // Bind the click event of each element to a function
   document.querySelectorAll('.card').forEach(card => {
     card.addEventListener('click', () => {
       // TODO: write some code here
-      console.log(`Card clicked: ${card}`);
+      let card1;
+      let card2;
+      card.querySelectorAll('.card > div').forEach(div => {
+        console.log(div);
+        div.classList.toggle('front');
+        div.classList.toggle('back');
+      });
+      memoryGame.pickedCards.push(card);
+      if (memoryGame.pickedCards.length === 2) {
+        card1 = memoryGame.pickedCards[0].getAttribute('data-card-name');
+        card2 = memoryGame.pickedCards[1].getAttribute('data-card-name');
+        //console.log('2 cards');
+
+        if (memoryGame.checkIfPair(card1, card2)) {
+          memoryGame.pickedCards[0].classList.add('empty');
+          memoryGame.pickedCards[1].classList.add('empty');
+          memoryGame.pickedCards = [];
+        } else {
+          console.log(memoryGame.pickedCards);
+          setTimeout(() => {
+            console.log(memoryGame.pickedCards);
+            memoryGame.pickedCards.forEach(card => {
```

Comment:
This code looks the same as the code starting on line 50. Could the callback be stored in a function? Then the "toggle" function can be called twice.